poc/vdaf_prio3.py

```diff
@@ -483,6 +509,41 @@ class Prio3HistogramWithLength(Prio3Histogram):
         return Prio3HistogramWithLength
 
 
+class Prio3SumVecWithMultiproof(Prio3SumVec):
+    ID = 0xFFFFFFFF  # TBD
+
+    # Operational parameters.
+    test_vec_name = 'Prio3SumVecWithMultiproof'
+
+    @staticmethod
+    def is_recommended(num_proofs: Unsigned, field: FftField) -> bool:
+        # To be confirmed
+        if field == Field64:
+            return num_proofs >= 2
+        elif field == Field96:
+            return num_proofs >= 2
+        elif field == Field128:
+            return num_proofs >= 1
+        return False
+
+    @classmethod
+    def with_params(cls,
+                    length: Unsigned,
+                    bits: Unsigned,
+                    chunk_length: Unsigned,
+                    num_proofs: Unsigned,
```

Comment:
[One feedback](https://github.com/cfrg/draft-irtf-cfrg-vdaf/issues/177#issuecomment-1739646213) we got in issue 177 is the algorithm identifier needs to commit to the choice of field and number of proofs. I think that means one code point needs to map to a particular choice of field and number of proofs. The current Prio3 does so by including the algorithm identifier in the domain separation tag.